src/Yesod/Markdown.hs

```diff
@@ -27,6 +27,7 @@ module Yesod.Markdown
   -- * Option sets
   , yesodDefaultWriterOptions
   , yesodDefaultReaderOptions
+  , yesodDefaultOptionExtensions
```

Comment:
Can you call this `yesodDefaultExtensions` to be consistent with the other function that are all `yesodDefault{PandocType}`

Reply:
Sounds good.  I'll make a change.

Reply:
I've changed the name of `yesodDefaultOptionExtensions` to just `yesodDefaultExtensions`.